xcode/CUPS.xcodeproj/project.pbxproj

```diff
@@ -11582,6 +11592,7 @@
 					.,
 					..,
 				);
+				LLVM_LTO = YES;
```

Comment:
This doesn't hurt but doesn't help - the Xcode project is more to make it easier to do Clang static analysis of the code than to build it for macOS...